portfolio/src/main/webapp/script.js

```diff
@@ -21,11 +21,24 @@
  * Fetches hello from the server and adds it to the page.
  */
 async function addHello() {
-  fetch('/data').then((response) => response.json()).then((hello) => {
-    document.getElementById('hello-container').innerText = hello;
+  fetch('/data').then((response) => response.json()).then((greetings) => {
+    const greetingListElement = document.getElementById('hello-container');
+    greetingListElement.innerHTML = '';
+    console.log(greetings);
+    for (const message of greetings) {
+      console.log(message);
```

Comment:
nit: You should remove the console.log() statements.

Reply:
Oh oops!